src/extension/utils.ts

```diff
@@ -106,8 +111,8 @@ export function getKey(runningCell: vscode.TextDocument): keyof typeof executor
  * treat cells like like a series of individual commands
  * which need to be executed in sequence
  */
-export function getCmdShellSeq(cellText: string, os: string): string {
-  const trimmed = cellText
+export function getCmdSeq(cellText: string): string[] {
```

Comment:
Once the TUI is using the Runner to execute commands we should consider GRPC exposing APIs to prep an input as a sequence of commands so both Ext/TUI plausible work the same way. Let's make a ticket for this if we haven't already.

Reply:
https://github.com/stateful/runme/issues/157